src/pages/Contact.js

```diff
@@ -1,10 +1,35 @@
 import React from 'react';
 import '../App.js';
+import ContactForm from '../components/ContactForm.js';
+import './Contact.css';
+import { Link } from 'react-router-dom';
 
 export default function Contact() {
   return (
-    <h1 className='contact'>
-      Contact me from here 
-    </h1>
+    <div>
+      <div>
+        <h1>Contact</h1>
+        <p>Want to work together on a project?</p>
+      </div>
+      <div>
+        <ContactForm></ContactForm>
```

Comment:
this is more of a preference thing, but if an element is provided no children, then the component can be used in a self closing manner
```suggestion
        <ContactForm />
```